internal/injector/injector.go

```diff
@@ -58,7 +59,6 @@ type (
 func New(pkgDir string, opts Options) (*Injector, error) {
 	fileset := token.NewFileSet()
 	cfg := &packages.Config{
-		Dir:  pkgDir,
```

Comment:
This actually causes issues; because this directory is the one from which the `go.mod` file will be resolved, ensuring modules appropriately are resolved to the correct version. The `go` toolchain normally does this in the context of `$PWD` and so we should be replicating the same behavior, or else we'll get surprising results.